docs/reST/ref/rect.rst

```diff
@@ -348,6 +348,52 @@
       with the Rect. If no intersecting rectangles are found, an empty list is
       returned.
 
+      Not only Rects are valid arguments, but these are all valid calls:
+
+      .. code-block:: python
+          :linenos:
+    
+          Rect = pygame.Rect
+          r = Rect(0, 0, 10, 10)
+
+          list_of_rects = [Rect(1,1,1,1), Rect(2,2,2,2)]
+          indices0 = r.collidelistall(list_of_rects)
+    
+          list_of_lists = [[1,1,1,1], [2,2,2,2]]
+          indices1 = r.collidelistall(list_of_lists)
+    
+          list_of_tuples = [(1,1,1,1), (2,2,2,2)]
+          indices2 = r.collidelistall(list_of_tuples)
+    
+          list_of_double_tuples = [((1,1), (1,1)), ((2,2), (2,2))]
+          indices3 = r.collidelistall(list_of_double_tuples)
+    
+          class ObjectWithRectAttribute(object):
```

Comment:
You could remove the (object) here because of Python3. Same for the others

Reply:
sure, old habits, thanks :)